utils.py

```diff
@@ -453,3 +454,38 @@ def get_all_language_codes_and_names():
 def unescape_encoded_uri_component(escaped_string):
     """Unescape a string that is encoded with encodeURIComponent."""
     return urllib.unquote(escaped_string).decode('utf-8')
+
+
+CACHE_SLUG = None
+def get_cache_slug():
```

Comment:
I'm not entirely convinced that this needs to be a separate function. You might be able to combine it with get_asset_dir_prefix() (and, in the process, get rid of the notion of a cache slug for dev).